src/main/java/org/symphonyoss/integration/webhook/github/parser/v2/GithubMetadataParser.java

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright 2016-2017 Symphony Integrations - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.integration.webhook.github.parser.v2;
+
+import static org.symphonyoss.integration.parser.ParserUtils.MESSAGEML_LINEBREAK;
+import static org.symphonyoss.integration.parser.ParserUtils.buildEncodedUrl;
+import static org.symphonyoss.integration.webhook.github.GithubEventConstants
+    .GITHUB_HEADER_EVENT_NAME;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.EVENT_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.ICON_URL_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.LOGIN_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.NAME_TAG;
+import static org.symphonyoss.integration.webhook.github.GithubEventTags.URL_TAG;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.symphonyoss.integration.model.message.Message;
+import org.symphonyoss.integration.model.yaml.IntegrationProperties;
+import org.symphonyoss.integration.service.UserService;
+import org.symphonyoss.integration.webhook.github.parser.GithubParser;
+import org.symphonyoss.integration.webhook.github.parser.GithubParserException;
+import org.symphonyoss.integration.webhook.github.parser.GithubParserUtils;
+import org.symphonyoss.integration.webhook.parser.metadata.MetadataParser;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.PostConstruct;
+import javax.ws.rs.ProcessingException;
+
+/**
+ * Abstract Github parser responsible to augment the Github input data querying the user API and
+ * pre-processing the input data.
+ * Created by campidelli on 02/05/17.
+ */
+public abstract class GithubMetadataParser extends MetadataParser implements GithubParser {
+
+  private IntegrationProperties integrationProperties;
```

Comment:
I think it's more readable if you start the attributes declaration with the static fields.